### PR TITLE
Modified setImmediate polyfill to fix IE/Edge bug

### PR DESCRIPTION
**Pull request by JeremyRann**
https://github.com/digitalbazaar/forge/issues/385
I tested this using the automated tests in node, Internet Explorer, Edge, Firefox, and Google Chrome. The automated tests did indeed fail in Edge and Internet Explorer on the master branch, but this commit fixes those tests with no regressions that I can see.

### CODE REVIEW

File: js/util.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ var util = forge.util = forge.util || {};
 
   // polyfill nextTick with native setImmediate
   if(typeof setImmediate === 'function') {
-    util.setImmediate = setImmediate;
+    util.setImmediate = function () { return setImmediate.apply(undefined, arguments); };
```
Comment (dlongley) [inline js/util.js]: Minor style nit:
` function ()` => ` function()`

**Comment by dlongley** [comment]
Hi @JeremyRann,

Can you fix the minor style nit, squash, and rebase on master? Thanks!

**Comment by JeremyRann** [comment]
I'm afraid I've gotten very confused attempting to squash and rebase; I can't seem to get this pull request down to one commit readily, and I've even managed to rewrite history in a way that orphaned your previous comment. Should I continue trying to squash this down?

**Comment by dlongley** [comment]
@JeremyRann, no, that's ok. I can do the rebasing and squashing.

**Comment by dlongley** [comment]
Landed as 2bc35135cb561ba7255b2386f27d953c3fca9492.

**Comment by dlongley** [comment]
Thanks, @JeremyRann!

**Pull request closed without merging**